test/role.test.js

```diff
@@ -511,6 +554,24 @@ describe('role model', function() {
       });
     });
 
+    it('should resolve user by id using custom user principalType', function(done) {
+      ACL.resolvePrincipal('User', user.id, function(err, u) {
+        if (err) return done(err);
+
+        expect(u.id).to.eql(user.id);
+
+        done();
```

Comment:
Nitpick: in cases like this, we usually don't pad with empty lines, as they add too much visual overhead (at least IMO).

```js
if (err) return done(err);
expect(u.id).to.eql(user.id);
done();
```